core-bundle/src/Cache/ContaoCacheWarmer.php

```diff
@@ -155,12 +159,16 @@ private function generateLanguageCache(string $cacheDir): void
 
                 $processed[] = $name;
 
-                $subfiles = $this->finder->findIn('languages/'.$language)->files()->name('/^'.$name.'\.(php|xlf)$/');
+                $subfiles = $this->finder
+                    ->findIn(Path::join('languages', $language))
+                    ->files()
+                    ->name('/^'.$name.'\.(php|xlf)$/')
+                ;
 
                 try {
                     $dumper->dump(
                         iterator_to_array($subfiles),
-                        sprintf('languages/%s/%s.php', $language, $name),
+                        Path::join('languages', $language, "$name.php"),
```

Comment:
```suggestion
                        Path::join('languages', $language, $name.'.php'),
```

Reply:
Is there any reason why you'd want to have concatenation instead of interpolation? 
(besides personal preference which is fine oc :smile: )

Reply:
Concatenation is more performant than interpolation.

Reply:
That's not true anymore for PHP7.

see https://blog.blackfire.io/php-7-performance-improvements-encapsed-strings-optimization.html

Reply:
Interesting. Based on this information, we should even prefer interpolation over concatenation?

Reply:
Yes, I think so. But imo readability should come first as the performance difference is negligible in most cases.